src/main/java/de/tum/in/www1/artemis/service/connectors/pyris/dto/status/IngestionState.java

```diff
@@ -0,0 +1,8 @@
+package de.tum.in.www1.artemis.service.connectors.pyris.dto.status;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public enum IngestionState {
```

Comment:
I'd expect this in the `domain` package. Why did you put it to the DTOs instead?